src/games/strategy/engine/chat/ChatMessage.java

```diff
@@ -0,0 +1,26 @@
+package games.strategy.engine.chat;
+
+
+public class ChatMessage {
+	private final String message;
+	private final String from;
+	private final boolean isMessage;
```

Comment:
Not sure if this is a good variable name change. I'm guessing that boolean is referring to whether the message came from the current client (not whether it is a message). Probably should be "isMyMessage".